Gemfile

```diff
@@ -45,6 +45,7 @@ gem 'warden-basic_auth', '~> 0.2.1'
 gem 'rails_autolink', '~> 1.1.6'
 gem "will_paginate", '~> 3.0'
 gem "acts_as_list", "~> 0.3.0"
+gem 'acts_as_countable', git: "https://github.com/finnlabs/acts_as_countable.git"
```

Comment:
I don't think I can set a specific version lock from the git repository, because it simply uses whatever is behind that refs gemspec.

We can probably fix it to the current commit (happens more often in our Gemfile) and update that explicitly?